org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/junit/JUnitFragment.xtend

```diff
@@ -72,7 +73,9 @@ class JUnitFragment extends AbstractStubGeneratingFragment {
 	protected def String getXbaseUiTestingPackage() {
```

Comment:
do dont see any way how to make this transition nicely. the fragment is very wird in the test plugin regards